packages/components/src/components/primitives/Typography/Typography.stories.tsx

```diff
@@ -1,91 +1,40 @@
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Headline2</p>
-      <Typography tag="h1" type={TypographyTypes.Headline2}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Headline3</p>
-      <Typography tag="h1" type={TypographyTypes.Headline3}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Headline4</p>
-      <Typography tag="h1" type={TypographyTypes.Headline4}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Headline5</p>
-      <Typography tag="h1" type={TypographyTypes.Headline5}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Headline6</p>
-      <Typography tag="h1" type={TypographyTypes.Headline6}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-  </div>
-);
+enum tagOptions {
+  h1 = 'h1',
+  h2 = 'h2',
+  h3 = 'h3',
+  h4 = 'h4',
+  h5 = 'h5',
+  h6 = 'h6',
+  p = 'p',
+  span = 'span',
+  label = 'label',
+}
 
-export const Copy = () => (
-  <div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Body</p>
-      <Typography tag="p" type={TypographyTypes.Body}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Detail</p>
-      <Typography tag="p" type={TypographyTypes.Detail}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Eyebrow</p>
-      <Typography tag="p" type={TypographyTypes.Eyebrow}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Form</p>
-      <Typography tag="p" type={TypographyTypes.Form}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Button</p>
-      <Typography tag="p" type={TypographyTypes.Button}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-    <div className="flex flex-row items-center my-4">
-      <p className="mr-4">Link</p>
-      <Typography tag="p" type={TypographyTypes.Link}>
-        The quick brown fox jumps over the lazy dog.
-      </Typography>
-    </div>
-  </div>
+export const Typographies = () => (
```

Comment:
I like this as an interactive component to play with but not sure about removing the previous one, basically because having all options allows us to detect changes using chromatics

Reply:
Let's keep both then.